rb-inotify.gemspec

```diff
@@ -16,11 +18,11 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
-  
-  spec.required_ruby_version = '>= 2.0.0'
-  
-  spec.add_dependency "ffi", "~> 1.0"
-  
+
+  spec.required_ruby_version = '>= 0'
+
+  spec.add_dependency 'ffi', '>= 0.5.0', '< 2'
```

Comment:
I referred `rb-inotify`'s dependency condition.
https://rubygems.org/gems/rb-inotify/versions/0.9.8
Also I added `< 2`, because if we do not add it, we will get warning message when `gem build *.gemspec`.